src/app/Main/Body/QueryPage/Results/QueryPage-ResultData.component.tsx

```diff
@@ -90,36 +103,49 @@ export function QueryPageTableRestultComponent(
     );
 
     return (
-        <div css={dataStyle}>
-            <TableMainHeaderComponent
-                isTitle={props.displayRowCol.isCol}
-                isRow={false}
-                height={tableHeight}
-                width={headerWidth}
-                text={colMainHeader}
-            />
-            <div css={tableDivStyle}>
-                <TableMainHeaderComponent
-                    isTitle={props.displayRowCol.isRow}
-                    isRow
-                    height={tableHeight}
-                    width={headerWidth}
-                    text={rowMainHeader}
-                />
-                <div
-                    css={dataTableStyle}
-                    ref={(node: HTMLElement | null) => div(node, "totalWidth")}
-                >
-                    <ResultsTable
-                        allIsolates={table.statisticData}
-                        isIsolates={isIsolates}
-                        columnAttributes={props.columnAttributes}
-                        getSize={div}
-                        isRowNotCol={isRowNotCol}
-                        isRowAndCol={isRowAndCol}
+        <Accordion defaultExpanded>

```

Comment:
- so you are using the accordion multiple times in the app you could write a shared wrapper component for it

Reply:
Good idea. I made an extra ticket for that.